!Command
positional: []
named: []
command:
- elasticluster
parent:
subcommands: []
help_flag:
usage_flag:
version_flag:
help_text: "Traceback (most recent call last):\n  File \"/tmp/tmp9nw_qdw5/bin/elasticluster\"\
  , line 6, in <module>\n    from pkg_resources import load_entry_point\n  File \"\
  /tmp/tmp9nw_qdw5/lib/python2.7/site-packages/pkg_resources/__init__.py\", line 3251,\
  \ in <module>\n    @_call_aside\n  File \"/tmp/tmp9nw_qdw5/lib/python2.7/site-packages/pkg_resources/__init__.py\"\
  , line 3235, in _call_aside\n    f(*args, **kwargs)\n  File \"/tmp/tmp9nw_qdw5/lib/python2.7/site-packages/pkg_resources/__init__.py\"\
  , line 3264, in _initialize_master_working_set\n    working_set = WorkingSet._build_master()\n\
  \  File \"/tmp/tmp9nw_qdw5/lib/python2.7/site-packages/pkg_resources/__init__.py\"\
  , line 583, in _build_master\n    ws.require(__requires__)\n  File \"/tmp/tmp9nw_qdw5/lib/python2.7/site-packages/pkg_resources/__init__.py\"\
  , line 900, in require\n    needed = self.resolve(parse_requirements(requirements))\n\
  \  File \"/tmp/tmp9nw_qdw5/lib/python2.7/site-packages/pkg_resources/__init__.py\"\
  , line 786, in resolve\n    raise DistributionNotFound(req, requirers)\npkg_resources.DistributionNotFound:\
  \ The 'python-novaclient' distribution was not found and is required by elasticluster\n"
generated_using:
- --help
